cpp/arcticdb/pipeline/value.hpp

```diff
@@ -208,4 +208,27 @@ inline std::optional<std::string> ascii_to_padded_utf32(std::string_view str, si
     return rv;
 }
 
-} //namespace arcticdb
\ No newline at end of file
+} //namespace arcticdb
+
+namespace fmt {
+template<>
+struct formatter<arcticdb::Value> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    auto format(const arcticdb::Value& value, FormatContext& ctx) const {
+        if (is_sequence_type(value.type().data_type())) {
+            fmt::format_to(ctx.out(), "\"{}\"", std::string(*value.str_data(), value.len()));
+        }
+        else {
+            arcticdb::entity::details::visit_type(value.data_type_, [&](auto val_tag) {
```

Comment:
This is going to introduce an extra visitation over all of the types, but everywhere this is called is already inside a visitation over the value's type